### PR TITLE
[R] fix model_diagnostics with matrix data

### LINKED ISSUE #355: `model_diagnostics` is not working when data is a matrix
For XGBoost models the `data` is usually of type matrix.
In such cases `model_diagnostics` gives a WARNING due to
```
results$y <- explainer$y
```

The `$` works for data.frames.
Replace with `cbind` is possible.

### PR DESCRIPTION
**Pull request by hbaniecki**
fixes #355

### CODE REVIEW

File: R/model_diagnostics.R [hunk 1]
```diff
@@ -45,9 +46,9 @@ model_diagnostics <-  function(explainer, variables = NULL, ...) {
   # if variables = NULL then all variables are added
   # otherwise only selected
   if (is.null(variables)) {
-    results <- explainer$data
+    results <- as.data.frame(explainer$data)
```
Comment (pbiecek) [inline R/model_diagnostics.R:49]: do we need to covert it to data.frame?
would it be simple to use `rbind()` to keep the class of an object?

Comment (hbaniecki) [reply R/model_diagnostics.R:49]: At first, I used `cbind` in 10 places to add these columns. But then, don't we want the result of the explanation to be `data.frame`? It is used by plot etc.

File: DESCRIPTION [hunk 2]
```diff
@@ -33,5 +33,5 @@ Suggests:
     ranger,
     testthat,
     methods
-URL: https://ModelOriented.github.io/DALEX/, https://github.com/ModelOriented/DALEX
+URL: https://dalex.drwhy.ai, https://ModelOriented.github.io/DALEX/
```
[the remark this answers is not in the document]
Comment (hbaniecki) [reply DESCRIPTION:36]: Sure. Which 2 urls should I keep? Or all 3 of them?

**Review by pbiecek — APPROVED** [review]
ok, thanks

**Pull request merged**